src/substrate/types.ts

```diff
@@ -138,10 +138,10 @@ interface IEvent {
  */
 export interface INewSession extends IEvent {
   kind: EventKind.NewSession
-  validators: Vec<ValidatorId>;
-  exposure: Vec<Exposure>; // @mir-nawaz lmk if there is an issue with this
+  validators: Array<ValidatorId>;
+  exposure: Array<Exposure>;
```

Comment:
I'm still a little worried about our usage of this Exposure struct. It will work for now, but if we ever turn chain events into a queue producer, then this will fail (due to serialization concerns).

Reply:
How [this](https://github.com/hicommonwealth/chain-events/pull/15#discussion_r479589635) works

Reply:
Nope -- we still need to unwrap the type.

Reply:
Yea, I agree with @jnaviask. We should be deserializing these types and reserializing them in CW repo when we pull from the DB. You can even just use `Exposure.toHuman()` right?